gtk/transmission-gtk.metainfo.xml.in

```diff
@@ -29,6 +29,14 @@ Copyright 2017 Endless Mobile, Inc.
     </p>
   </description>
   <url type="homepage">https://transmissionbt.com/</url>
+  <url type="bugtracker">https://github.com/transmission/transmission/issues</url>
+  <url type="faq">https://transmissionbt.com/help/gtk/4.0x/html/FAQ.html</url>
+  <url type="help">https://transmissionbt.com/help/gtk/4.0x/index.html</url>
+  <url type="donation">https://transmissionbt.com/donate</url>
+  <url type="translate">https://explore.transifex.com/transmissionbt/transmissionbt/</url>
+  <url type="contact">https://forum.transmissionbt.com/</url>
```

Comment:
Not sure what "contact" means here, but the link points to user forum devs rarely pay attention to currently.

Reply:
https://github.com/transmission/transmission/discussions might be better

Reply:
The commit message links to [the definition of the different URL types in the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url).

> **contact**
> URLs of this type should allow the user to contact the developer.
> 
> This could for example be an HTTPS URL to an online form or a page describing how to contact the developer.

I can change it to the GitHub discussions page. Just so you know, the same forum link is used as the [ARPHELPLINK](https://learn.microsoft.com/en-us/windows/win32/msi/arphelplink) on [Windows](https://github.com/wjt/transmission/blob/e86a0693d494d54fb9ff8a68bc2b9c6d6fd4d30a/dist/msi/Transmission.wxs#L103), [in the readme](https://github.com/wjt/transmission/blob/e86a0693d494d54fb9ff8a68bc2b9c6d6fd4d30a/docs/README.md#L10), [in the macOS client](https://github.com/wjt/transmission/blob/e86a0693d494d54fb9ff8a68bc2b9c6d6fd4d30a/macosx/Controller.mm#L123), and a number of other places with similar meaning.